sandbox/lua/filters/procstat.lua

```diff
@@ -0,0 +1,213 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+
+Calculates deltas in /proc/stat data.
+Also emits CPU percentage utilization information.
+
+Config:
+
+- whitelist (string, optional, default nil)
+    Only proccess fields that fit the pattern, defaults to match all
```

Comment:
Spelling: "process" instead of "proccess"; there are a couple others like this in the file too.